modular_zubbers/modules/customization/modules/language/common.dm

```diff
@@ -0,0 +1,13 @@
+/datum/language/common
+	name = "Galactic Common"
+	desc = "Common, as it's usually referred to in The Federation, is an amalgamation of languages spoken in the Core, Sol, Theasperri, and Tiziria, handcrafted by all the founding states to form a comprehensive and familiar language for nearly all citizens in the Federation to learn and communicate with one another. It is not uncommon for people to speak both Common and their homeland's tongue, as Galactic Common, by design, shares many structures and quirks with many languages, almost to the extent that speaking Galactic Common is speaking 20 different languages simultaneously."
+	space_chance = 60
+	syllables = list(
+		"at", "bo", "cre", "del", "ei", "fir", "guh", "ho", "ill", "jo", "kat", "le", "mon", "ne", "op", "po", "que", "ro", "set", "tin", "um", "ven", "wir", "xa", "yo", "zet",
+		"ao", "bar", "co", "don", "ef", "fo", "gon", "hi", "io", "jit", "kir", "lol", "mit", "no", "oh", "pa", "quo", "rex", "so", "ta", "ur", "va", "wa", "xi", "ye", "zat",
+		"ah", "be", "ch", "da", "eu", "fu", "ga", "ha", "in", "ja", "ko", "li", "ma", "nya", "oi", "per", "qe", "ri", "sex", "tic", "uo", "vo", "wu", "xe", "yr", "ze",
+		"'al", "'bel", "'cal", "'do", "'el", "'feb", "'gol", "'hir", "'ib", "'jah", "'kil", "'lo", "'mo", "'no", "'ol", "'pek", "'qre", "'rib", "'sif", "'tal", "'ui", "'vik", "'wo", "'xin", "'yef", "'zak",
+		"ap", "br", "cze", "di", "eck", "fu", "gup", "he", "ir", "je", "ka", "la", "mer", "na", "os", "pi", "qux", "rum", "sin", "tum", "ut", "vye", "weh", "xje", "yoh", "zog"
+	)
+	icon_state = "solcommon"
+	icon = 'modular_skyrat/master_files/icons/misc/language.dmi'
```

Comment:
```suggestion
```

Reply:
This removes changes actually changed to the file. I removed the repetitive changes and re-added the syllables list (which was changed)